gbols/fast-food
# #160478080 Endpoint to place a new order

**What does this PR do?**
 It posts a new order.

**Description of Task to be completed?**
- create an endpoint to update status of an order.

**How should this be manually tested?**
- pull from this branch `ft-post-order-160478080` into a new branch.
- run `npm install` to install all the requirements.
- endpoint should look like this `http://localhost:3000/api/v1/orders`
- run npm test to run the test.

**What are the relevant pivotal tracker stories?**
#160478080 user should be able post a food order.

**Postman Screenshots**

![screenshot from 2018-09-13 15-10-03](https://user-images.githubusercontent.com/27724644/45494532-2ed59b00-b769-11e8-82ef-9439d52342f6.png)
![screenshot from 2018-09-13 15-13-36](https://user-images.githubusercontent.com/27724644/45494538-31d08b80-b769-11e8-9c1a-4801b0c5c496.png)
